tests/test_api.py

```diff
@@ -2313,9 +2304,12 @@ def test_reactor_config_secret(self):
 
         flexmock(OSBS, _create_build_config_and_build=request_as_response)
 
-        req = osbs_obj.create_build(target=TEST_TARGET,
-                                    architecture=TEST_ARCH,
-                                    **REQUIRED_BUILD_ARGS)
+        kwargs = copy.deepcopy(REQUIRED_BUILD_ARGS)
+        kwargs.update({
+            'build_type': 'orchestrator',
+            'reactor_config_override': {'required_secrets': ['mysecret']},
+        })
+        req = osbs_obj.create_build(target=TEST_TARGET, architecture=TEST_ARCH, **kwargs)
```

Comment:
Is it necessary to copy `REQUIRED_BUILD_ARGS` here? Maybe `build_type` and `reactor_config_override` could just be passed to `osbs_obj.create_build` directly.

Reply:
Done, thanks!